File: V1P1/V1P1.go
/*

 El algorito toma una temperatura Celcius y la convierte en Fahnereit y Kelvin, y los retorna.

*/

package main

import "fmt"

func main() {

	var c float64

	print(": ")
	fmt.Scanln(&c)

	f := (c * 9 / 5) + 32 // Conversion a Fahnereit
	k := c + 273.15       // Conversion a Kelvin

	fmt.Printf("Celsius: %f, Fahrenheit: %f, Kelvin: %f\n", c, f, k)
}

File: V1P1/V1P1.rs
use std::io;

fn main() {
    let mut entrada = String::new();

    io::stdin().read_line(&mut entrada).unwrap();

    let c: f64 = entrada.trim().parse().unwrap();

    let f = (c * 9.0 / 5.0) + 32.0;
    let k = c + 273.15;

    println!("Celsius: {}, Fahrenheit: {}, Kelvin: {}", c, f, k);
}
